app/models/agents/liquid_output_agent.rb

```diff
@@ -0,0 +1,131 @@
+module Agents
+  class LiquidOutputAgent < Agent
+    include WebRequestConcern
+    include FormConfigurable
+
+    cannot_be_scheduled!
+
+    description  do
+      <<-MD
+        The Liquid Output Agent outputs events through a Liquid template you provide.  Use it to create a HTML page, or a json feed, or anything else that can be rendered as a string from your stream of Huginn data. 
+
+        This Agent will output data at:
+
+        `https://#{ENV['DOMAIN']}#{Rails.application.routes.url_helpers.web_requests_path(agent_id: ':id', user_id: user_id, secret: ':secret', format: :any_extension)}`
+
+        where `:secret` is thel secret specified in your options.  You can use any extension you wish.
+
+        Options:
+
+          * `secrets` - An array of tokens that the requestor must provide for light-weight authentication.
```

Comment:
Well, it's confusing when I have a plural word describing a singular concept.  I made a checkbox to fix that, and it's been resolved.  I missed this spot in the documentation, though.

I'm going to change the view like the data output agent to expose the URLs that become available when these options are used.  I think that should clear things up.